features/step_definitions/common_steps.rb

```diff
@@ -344,7 +344,7 @@
   edit_link.click
 end
 
-When /I click a summary table '(.*)' (link|button) for '(.*)'$/ do |link_name, elem_type, field_to_edit|
+When /I click a summary table #{MAYBE_VAR} (link|button) for #{MAYBE_VAR}$/ do |link_name, elem_type, field_to_edit|
```

Comment:
Possible symptom of me not looking at the functional tests enough, but I always end up having to remind myself what extra `MAYBE_VAR` gives us and why it is good. Perhaps a quick sentence in the README pointing to the `transforms.rb` would be useful (doesn't have to be in this PR?).